### DISCUSSION TITLE
Osu Lazer - Sort by Favorited Maps

### DISCUSSION THREAD

**Discussion by Pumperknuckle** [ppy/osu#25054] [Ideas] [answered]
The stable client allows users to Group by Favorites while the Lazer client has no such functionality.

I have been using Lazer exclusively for months now and it is an improvement in almost every way except for the grouping/sorting features, with the most difficult exclusion being the ability to filter by favorites. I have been playing osu! for years and have 14,000+ beatmaps. Filtering maps to only the ones I favorite was my main way to scroll through my beatmaps and it feels like an obvious feature to have included. I love the "Group by" and "Sort By" distinction. I'm not asking for it to come back, but

**I would like to see a way to filter maps by favorites.**

If this is already possible through an advanced search, then I would instead suggest an easier way IN-GAME to learn all the search features. Adding all of my Favorites to a collection would be tedious as it would be ~500 maps that I didn't previously have to create and maintain.

Thank your for your time and work :)

**Comment by Rekunan** [comment, marked as answer]
Duplicate of #22798
